middleware/redis_store.js

```diff
@@ -0,0 +1,119 @@
+var RedisPool = require('sol-redis-pool');
+
+function redisStore(args) {
```

Comment:
@lourou looks like you wrote your own Redis store, why didn't you use https://www.npmjs.com/package/cache-manager-redis ?
Also, you should connect to Redis by the env variable `REDIS_URI`, like in [`routes/index.js` here](https://github.com/MeetMangrove/mangrove-website/pull/38/files#diff-bb503f3d9047bcd4a5730559f1e16560R6) or it wont work on Heroku! Let me know if that's not clear!